src/Tribe/Integrations/WPML/Language_Switcher.php

```diff
@@ -44,7 +44,7 @@ public function filter_icl_ls_languages( array $languages = array() ) {
 		$root_folder = parse_url( home_url(), PHP_URL_PATH );
 		$request_uri = $_SERVER['REQUEST_URI'];
 
-		if ( ! empty( $root_folder ) ) {
+		if ( $root_folder !== '/' ) {
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: fd25b2a7c221b94eaad42957ea9526f7</sub>